images/scripts/debian.setup

```diff
@@ -86,6 +92,10 @@ case "$RELEASE" in
         IPA_CLIENT_PACKAGES="${IPA_CLIENT_PACKAGES/freeipa-client /}"
 esac
 
+cat <<EOF | sudo tee /etc/apt/sources.list.d/dbgsym.list
+deb http://debug.mirrors.debian.org/debian-debug/ ${RELEASE}-debug main
+EOF
```

Comment:
This is debian specific, and doesn't apply to the Ubuntu images. So they need to be guarded. Maybe easiest to move them into the `case` above?

Reply:
Oh good catch, I will. 
Yes, we have some crashes which are not naughted.

Reply:
Thanks! That works. Not approving yet, as you still need to push the rebuilds and test them, but :+1: